test/e2e/specs/adding-blocks.test.js

```diff
@@ -10,23 +10,9 @@ describe( 'adding blocks', () => {
 		await newPost();
 	} );
 
-	/**
-	 * Given a Puppeteer ElementHandle, clicks below its bounding box.
-	 *
-	 * @param {Puppeteer.ElementHandle} elementHandle Element handle.
-	 *
-	 * @return {Promise} Promise resolving when click occurs.
-	 */
-	async function clickBelow( elementHandle ) {
-		const box = await elementHandle.boundingBox();
-		const x = box.x + ( box.width / 2 );
-		const y = box.y + box.height + 1;
-		return page.mouse.click( x, y );
-	}
-
 	it( 'Should insert content using the placeholder and the regular inserter', async () => {
 		// Click below editor to focus last field (block appender)
-		await clickBelow( await page.$( '.editor-default-block-appender' ) );
+		await page.click( '.editor-writing-flow__click-redirect' );
```

Comment:
This refactor may have made the test less effective at covering the expected behavior: https://github.com/WordPress/gutenberg/pull/11168#pullrequestreview-169297243

Reply:
What's the best way to enhance this test again? CC: @gziolo who graciously helped me with this.

Reply:
Ooops, I wasn't aware of that :(